dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPExcTests.java

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jsp23.fat.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.logging.Logger;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.jsp23.fat.JSPUtils;
+import com.meterware.httpunit.GetMethodWebRequest;
+import com.meterware.httpunit.WebConversation;
+import com.meterware.httpunit.WebRequest;
+import com.meterware.httpunit.WebResponse;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Tests to execute on the jspServer that use HttpUnit/HttpClient
+ */
+
+@SkipForRepeat("CDI-2.0")
+@RunWith(FATRunner.class)
+public class JSPExcTests {
+    private static final Logger LOG = Logger.getLogger(JSPExcTests.class.getName());
+    private static final String TestServlet_APP_NAME = "TestServlet";
+
+    @Server("jspServletExceptionWrapServer")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+
+        ShrinkHelper.defaultDropinApp(server, TestServlet_APP_NAME + ".war",
+                                      "com.ibm.ws.jsp23.fat.testjsp23.beans",
+                                      "com.ibm.ws.jsp23.fat.testjsp23.servlets");
+
+        server.startServer(JSPExcTests.class.getSimpleName() + ".log");
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        // Stop the server, with expected exception
+        if (server != null && server.isStarted()) {
+            server.stopServer("SRVE0777E");
+        }
+    }
+
+    /**
+     * Test Exception in log following exception in dispatch forward
+     * call with donotcloseoutputonforwardforservleterror=true
+     *
+     * @throws Exception
+     *                       if something goes wrong
+     */
+    @Test
+    @Mode(TestMode.FULL)
```

Comment:
If all tests are FULL you can add the annotation at the class level rather than the method level